File: CompleteSolutions/betype.cpp
// https://quera.org/problemset/28948/
// writed with chatGPT python to cpp

#include <iostream>
#include <string>
using namespace std;

int main() {
    string s;
    cin >> s;

    int i = 0;
    while (i < s.length()) {
        if (s[i] == '=') {
            s.erase(i, 1);
            if (i > 0) {
                s.erase(i-1, 1);
            }
            // After removing two characters, decrease the index by 1
            i -= (i > 0)? 1:0;
        } else {
            i++;
        }
    }

    cout << s << endl;
    return 0;
}

File: CompleteSolutions/betype.py
# https://quera.org/problemset/28948/

s = list(input())

i = 0
while i < len(s):
    if s[i] == '=':
        s.pop(i)
        if i > 0:
            s.pop(i-1)
            i -= 1
        # After removing two characters, decrease the index by 1
        i -= 1 if i < 0 else 0
    else:
        i += 1

print(''.join(s))
